java/fundamentals/intro-to-java/compiling-and-running-java-code.md

```diff
@@ -22,32 +20,28 @@ inAlgoPool: false
 ---
 ## Content
 
-In the last insight, we talked about the `HelloWorld` class and its output but never mentioned how to get from one point to the other.
+Now that you've seen some Java code, you might be wondering how you can get that to run on your machine.
 
 Before the *Java Virtual Machine* can do its part and run the code, a *compiler*[1] is needed to get the *bytecode* understandable by the JVM[2].
 
-The **compiler** takes the Java code written by the programmer and outputs the *Java classes bytecode* (which is platform neutral).
-
-The bytecode then goes through the JVM which transforms it into low-level machine code. Here's a graphical representation of the process:
+This process is done by an *Integrated Development Environment* (IDE) automatically. If you're not using an IDE, both the compilation and interpretation can be run manually. 
 
-![jvm-mental-model](https://img.enkipro.com/a038da2671b98d52c795730e3a676198.png)
+In both cases, a Java compiler and the JVM have to be installed on the computer. You can get them by installing the *Java Development Kit* (JDK).
 
-This process is done by an *Integrated Development Environment* (IDE) automatically, or both the compilation and interpretation can be run manually. In both cases, a Java compiler and the JVM have to be installed on the computer. 
+> 💡 If you don't know how to install the JDK, check the Learn More section for instructions.
 
-You can get them by installing the *Java Development Kit* (JDK).
-
-Let's take the `HelloWorld.java` class for example. To compile it into `HelloWorld.class` file, you have to open a terminal in the folder containing the initial `.java` file and run:
-```
-$ javac HelloWorld.java
+Let's take a look at the previous `HelloJava.java` example[3]. To run it, you'll need to follow these two steps:
+1. Compile it to bytecode:
+```bash
+javac HelloWorld.java
```

Comment:
we could mention that javac = java compile ? not sure how accurate this is, but this analogy helped me remember the command

Reply:
added:
> 💡 Think of `javac` as **java compile**.